components/datadog/apps/fakeintake/fargateFakeintakeService.go

```diff
@@ -159,7 +248,7 @@ func fargateLinuxContainerDefinition(imageURL string) *ecs.TaskDefinitionContain
 		Environment: ecs.TaskDefinitionKeyValuePairArray{
 			ecs.TaskDefinitionKeyValuePairArgs{
 				Name:  pulumi.StringPtr("GOMEMLIMIT"),
-				Value: pulumi.StringPtr("768MiB"),
+				Value: pulumi.StringPtr("1536MiB"),
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
I was thinking that we might reduce OOM by filtering the stored payloads, either in space, including only interesting metric names, or in time, limiting retention to 5 minutes, which should be more than enough for the tests

Reply:
I think that we already have the payload retention limited in the fake intake.
So, if the flow of data remains more or less constant, the memory consumption of the fake intake should already stabilize after a period of time equal to the retention period.

The reason why I had to bump the memory is because, before this PR, the fake intake contained only metrics payloads. But with this fix, it now also contains logs payloads.
And the extra data made it be OOM killed again.

So, I had to increase the memory to make it able to store logs.